R/layouts.R

```diff
@@ -251,3 +251,27 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+#' @export
+#' @rdname flow_layout
```

Comment:
add roxygen documentation, eg see: https://roxygen2.r-lib.org/

Reply:
It also needs this example to be addded:

```r
if (interactive()) {
library(shiny.semantic)
ui <- semanticPage(flow_layout(
  numericInput("rows", "How many rows?", 5),
  selectInput("letter", "Which letter?", LETTERS),
  sliderInput("value", "What value?", 0, 100, 50)
 )
)
shinyApp(ui, server = function(input, output) { })
}
```

Reply:
Done.